app/assets/stylesheets/views/_guide.scss

```diff
@@ -21,4 +21,12 @@
       @include bold-24;
     }
   }
+
+  .task-list-sidebar {
+    margin-top: 0;
```

Comment:
Should this follow the pattern we've used for other components when it comes to top/bottom margins? Example: https://github.com/alphagov/government-frontend/blob/master/app/views/components/_notice.html.erb#L5

This means we don't have to override component styles in specific cases.